UnitedIncome/components
# TypeScript and React Hooks Hygen Generation

### Description
> Please provide a brief description of your changes below.

You can now `yarn generate` to generate `.tsx` files, along with TS types instead of PropTypes. I've also setup a defaultProps example.

Along with this I've also added a prompt for React Hooks as that's something we're also hoping to introduce soon. Didn't make sense to re-do the generator twice so I added it as an option if you choose for the component to be stateful.

### Testing Instructions
> What should be done in order to test this pull request, or re-create its effects.

* Run `yarn generate`.
* Try all of the options!
* Each example should pass the typescript linter.

### Comments
> Any additional comments that will aid with the review process.

Innovation sprint.
